x-pack/legacy/plugins/snapshot_restore/public/app/services/validation/validate_policy.ts

```diff
@@ -15,6 +15,12 @@ const isStringEmpty = (str: string | null): boolean => {
   return str ? !Boolean(str.trim()) : true;
 };
 
+const isSnapshotNameLowerCase = (str: string): boolean => {
+  const strExcludeDate =
+    str.substring(0, str.search('{')) + str.substring(str.search('}}') + 2, str.length);
```

Comment:
Is the double `}}` intended? At the moment the validation only passes when I add a trailing `}`. Resulting in:

<img width="1240" alt="Screenshot 2020-01-10 at 15 54 58" src="https://user-images.githubusercontent.com/8155004/72162156-b6fd9c00-33c1-11ea-8c1c-c01293d76c1e.png">

Reply:
A comment explaining this logic would also be great :)

Reply:
> Is the double `}}` intended? At the moment the validation only passes when I add a trailing `}`. Resulting in:
> 
> <img alt="Screenshot 2020-01-10 at 15 54 58" width="1240" src="https://user-images.githubusercontent.com/8155004/72162156-b6fd9c00-33c1-11ea-8c1c-c01293d76c1e.png">

The `}}` was intended. However, I think I've over analyzed [date format syntax](https://www.elastic.co/guide/en/elasticsearch/reference/current/date-math-index-names.html) and should just check for `}`  because all date format will have one `{` and `}` and not all have `}}`

> A comment explaining this logic would also be great :) - something about the what the string may contain or is expected to look like.

Will do, I will put it in the code